config.json.default

```diff
@@ -10,7 +10,9 @@
       "rpcEndpoint": "http://127.0.0.1:8545",
```

Comment:
Can we set the default RPC port as 43371 (while the p2p port will be 43370)?

Reply:
This is the endpoint for geth client - it's default port is 8545